<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Third</class>
 <widget class="QDialog" name="Third">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="profilePhoto">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>231</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="profileAge">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="profileName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="profileCity">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="profileHobbies">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>104</width>
     <height>70</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="likeButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Like</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dislikeButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Dislike</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>401</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>profilePhoto</zorder>
  <zorder>profileAge</zorder>
  <zorder>profileName</zorder>
  <zorder>profileCity</zorder>
  <zorder>profileHobbies</zorder>
  <zorder>likeButton</zorder>
  <zorder>dislikeButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
